sql/database.sql

```diff
@@ -3050,7 +3050,6 @@ CREATE TABLE `immunizations` (
   `ordering_provider` INT(11) DEFAULT NULL,
   `reason_code` varchar(31) DEFAULT NULL,
   `reason_description` text,
```

Comment:
above 2 lines missing the comments (figured you want them like the changes in upgrade script)